test/functionalTest/cases/1904_attr_value_type_none/1904_attr_value_type_none.test

```diff
@@ -0,0 +1,147 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PUT Attribute Value do not change type to 'none'
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. POST /v2/entities creating room
+# 02. PUT /v2/entities/room/attrs/temperature/value (forbidden chars)
+# 03. GET /v2/entities/room (type is not "none")
+
+echo '01. POST /v2/entities creating room'
+echo '==================================='
+payload=' {
+"temperature_2": { "type": "celsius",  "value": "34",  "metadata": { "very_hot_1": {"type": "alarm", "value": "hot"},  "very_hot_0": {"type": "alarm", "value": "hot"}}},
+"temperature_0": {
+       "type": "celsius",
+       "value": "34",
+       "metadata": {
+              "very_hot_1": {"type": "alarm", "value": "hot"},
+              "very_hot_0": {"type": "alarm", "value": "hot"}}},
+ "temperature_1": { "type": "celsius", "value": "34", "metadata": { "very_hot_1": {"type": "alarm", "value": "hot"}, "very_hot_0": {"type": "alarm", "value": "hot"}}},
+ "type": "house",
+ "id": "room_1"
+}
```

Comment:
Why such a long payload?
It makes it more difficult to understand the test.
If necessary for the test, then OK, of course, but ... is it?